### ISSUE TITLE
Improve error messages - "Language codes: en-US not supported"

### ISSUE THREAD

**Issue by Juliotati** [microsoft/msstore-cli#31]
After repeatedly this getting this error, I'm not sure what it means or what is even causing the error.
If possible please provide richer error messages or tips on what could have caused the error.

> Is this an issue on my project or from `msstore-cli` itself?

<img width="950" alt="Screenshot 2024-03-31 at 10 43 55" src="https://github.com/microsoft/msstore-cli/assets/59662912/7f7c2bd6-96db-40f9-9bcd-1ed7286704a6">

### Error json response
```json
{
  "code": "InvalidParameterValue",
  "data": [],
  "details": [],
  "message": "Language codes: en-US not supported",
  "source": "Ingestion Api",
  "target": "appListings"
}
```

**Comment by azchohfi** [comment]
We can't do much to improve the error messages, as these are coming from the API itself.
I believe this might be the listing language is not matching what you have in your msix.
Which command was issues and what is the type of project trying to be published?

**Comment by Juliotati** [comment]
> We can't do much to improve the error messages, as these are coming from the API itself. I believe this might be the listing language is not matching what you have in your msix. Which command was issues and what is the type of project trying to be published?

Thanks for the update and yes I was suspecting that the errors are something that would have to improved elsewhere.

Not sure if this is a normal error that appears on first releases are not.
The error came from my workflow step executed by:
```sh
msstore publish -i build/windows/x64/runner/Release/ --appId $PRODUCT_ID -v
```

But it seem that the package does get sent over to partner center for manual publishing after running `msstore reconfigure` even before or without running `msstore publish`.

**Comment by azchohfi** [comment]
This seems to be a regression in the API. 'en-US' should now be 'en-us'. We are pushing a small fix for this.

**Comment by azchohfi** [comment]
Should be fixed on v0.1.33.
